proto/slinky/mm2/v1/tx.proto

```diff
@@ -0,0 +1,80 @@
+syntax = "proto3";
+package slinky.mm2.v1;
+
+import "cosmos/msg/v1/msg.proto";
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+import "amino/amino.proto";
+import "slinky/mm2/v1/market.proto";
+import "slinky/mm2/v1/params.proto";
+
+option go_package = "github.com/skip-mev/slinky/x/mm2/types";
+
+// Msg is the message service for the x/marketmap module.
+service Msg {
```

Comment:
I think there's some stuff to add around permissions which I've mentioned in other PRs. Like add/remove market participant/mapper or whatever we call them.